app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,8 @@
+module CommentsHelper
+
```

Comment:
I would generally suggest that you follow this rule for new lines.

```
Never put a new line at the top of bottom of a scope
  So that you can see the structure of the tree

  Always put new lines between things on the same level with different responsibilities

  But if you have some things that are alike

  Like
    This
    And that
    Or this
  Y'know

  Then put them together to make it clear

  Same as the top, no empty lines at the bottom
Whitespace is our friend, be liberal
```